lib/src/lifecycle_module.dart

```diff
@@ -791,7 +791,8 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   /// A utility to logging LifecycleModule lifecycle events
   void _logLifecycleEvents(
       String logLabel, Stream<dynamic> lifecycleEventStream) {
-    listenToStream(lifecycleEventStream, (_) => _logger.fine(logLabel),
+
+    listenToStream(lifecycleEventStream, (_) { if (Uri.base.queryParameters['w_module.verbose'] == 'true') {_logger.fine(logLabel);}},
```

Comment:
Could we maybe do this as a global variable, sort of like what WSD does with its test mode, and then let applications determine how / when to set it?

Reply:
I like that idea too. Opt-in, right? e.g. `enableLifecycleLogging()`. What object would that live on?